src/shared/components/outboundLink/outboundLink.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactGA from 'react-ga';
+
+const OutboundLink = ({
+  href,
+  analyticsEventLabel,
+  children,
+  className,
+  style,
+}) => {
+  if (process.env.NODE_ENV === 'production') {
+    // https://operationcode.org is 25 characters - only show what follows
+    const location = window.location.href.slice(25);
```

Comment:
Can 25 be a const somewhere higher level that you pass in? Would make it easier to track down/update in once place if you need to reuse this value. 

Even better might be to use: `window.location.pathname` instead, if that accomplishes what you're looking for? More info on that here: https://stackoverflow.com/questions/736513/how-do-i-parse-a-url-into-hostname-and-path-in-javascript